src/constants.tsx

```diff
@@ -208,6 +208,8 @@ export const ROUTINE_PLAN = 'Routine';
 export type ROUTINE_PLAN = typeof ROUTINE_PLAN;
 export const CASE_TRIGGERED_PLAN = 'Case-triggered';
```

Comment:
Do you think that we can use the value from `FIReasons` in settings.ts instead of a new constant? 

Also, I think that this string should not be 'Case-triggered' but 'Case Triggered' (which is what is in the settings module, coming from this spec: https://docs.google.com/spreadsheets/d/1kGPgyF0eJdag19FFL1Oog2gdphdPtKobY4EQ0gMgtgw/edit#gid=1805975071).

Reply:
There's a few conditionals using this `CASE_TRIGGERED_PLAN`, am switching them to use `FIReasons[1]` instead, not sure if this will fix things that were silently broken before. I have confirmed that plans are using `Case Triggered` in the useContext array.